src/components/ListItem/ListItem.tsx

```diff
@@ -1,12 +1,13 @@
+import cx from "classnames";
+import { ComponentDefaultTestId, getTestId } from "../../tests/test-ids-utils";
 /* eslint-disable jsx-a11y/role-supports-aria-props,jsx-a11y/no-noninteractive-element-interactions */
 import React, { FC, forwardRef, ReactElement, useCallback, useRef } from "react";
-import cx from "classnames";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import { SIZES } from "../../constants/sizes";
 import { keyCodes } from "../../constants/keyCodes";
 import VibeComponentProps from "../../types/VibeComponentProps";
 import { NOOP } from "../../utils/function-utils";
-import "./ListItem.scss";
+import styles from "./ListItem.module.scss";
 
 const BEM_BASE_CLASS = "list-item";
```

Comment:
Here it would be better to remove these bem stuff completely and rename those class to `CSS_BASE_CLASS`, then the script would do the rest for u

Reply:
Next time :)